virt/virtual_machines/virtual_disks/virt-using-container-disks-with-vms.adoc

```diff
@@ -16,6 +16,7 @@ If you use large container disks, I/O traffic might increase, impacting worker n
 * xref:../../../nodes/nodes/nodes-nodes-garbage-collection.adoc#nodes-nodes-garbage-collection-configuring_nodes-nodes-configuring[Configuring garbage collection]
 ====
 
+include::modules/virt-golden-images.adoc[leveloffset=+1]
```

Comment:
Please update the introduction to refer to golden images so that the content has a better flow.